EXAMPLES/MATRIX_MARKET/arpackmm.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+includedir=@includedir@
+
+Name: arpackmm
+Description: Utility to test arpack with matrix market files
+Version: @PACKAGE_VERSION@
+URL: @PACKAGE_URL@
+Requires: arpack eigen3 >= 3.2
+Cflags: -I${includedir}/arpack
```

Comment:
To me, pc file is not necessary for `arpackmm`: it is a utility (binary/executable file). AFAIU, pc/cmake files are needed for libraries. @sylvestre may decide or not to add it in the install dir. You will never be able to "link to" `arpackmm`, but, you can use it at the command line prompt !

Reply:
I added it since arpackSolver.hpp is getting installed.  If some user needs that header, they can use arpackmm.h to get it.  If nobody needs it, let's not install arpackSolver.hpp and remove arpackmm.pc.

Reply:
Ah OK, didn't get that this way ! Indeed `arpackSolver.hpp` is installed if you want to compile it from your code : maybe rename `arpackmm.pc` to `arpackSolver.pc` ?

Reply:
I realize afterwards that : eigen flags are missing and needed to use `arpackSolver`, also `arpack.pc` is enough to use `arpackSolver`(all flags OK and eigen ones too)  => I think you can just kill `arpackmm.pc` !